Bottom: dcc9dc1439385a69bcf261312a2c43257a322b61
Top:    7e2ca760202a84c73ae729f0a274af833c0519f1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-10 17:20:47 -0500

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
